src/components/Start.tsx

```diff
@@ -1,58 +1,61 @@
+import React, { useState } from 'react'
 import { Categories, getAllTopics } from './Categories'
 import { useHistory } from 'react-router-dom'
-import React, { useState } from 'react'
 import { Score } from './Score'
 import { History } from 'history'
 import { prepareTopics } from '../services/question-service'
 
 export function Start() {
   const [time, setTime] = useState<string>('00:00')
-
+  const [name, setName] = useState<string>('')
   const [selectedTopics, setSelectedTopics] = useState<string[]>([])
   const history = useHistory<History>()
 
   const allTopics = getAllTopics().map((topic) => topic.name)
 
   return (
-    <>
+    <div className="mx-auto w-96 sm:w-screen font-retro">
       <div className="flex justify-between">
-        <div className="retro-font md:text-6xl sm:text-3xl text-gray-500 m-10">
+        <div className="m-10 text-gray-500 lg:text-4xl md:text-3xl sm:text-2xl">
           {time}
         </div>
         <Score score={'000000'} />
       </div>
-      <div className="App flex-col flex align-middle w-screen justify-center md:text-2xl">
+      <div className="flex flex-col justify-center align-middle md:text-2xl">
+        <div className="py-10 text-center text-gray-500 sm:text-2xl md:text-3xl">
+          Player name
+        </div>
         <input
           name="player"
-          placeholder="Player Name"
           type="text"
-          onFocus={(event) => (event.target.placeholder = '')}
-          onChange={(event) =>
-            event.target.placeholder?.length === 0
-              ? (event.target.placeholder = 'Player Name')
-              : void 0
+          autoFocus
+          value={name}
+          onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
```

Comment:
Nice! Need to figure out what I want to do with it though. Maybe display it on the result page...